README.md

```diff
@@ -28,11 +28,11 @@ can be downloaded.
 ## Overview Europace Reports
 The following reports can be called with this API:
 
-Name | Endpoint | Required Scope | File Type/Encoding | Content Description.
----- | ---- | ---- | :----: | ---
+Name | Endpoint | Required Scope | File Type/Encoding | Content Description.                                                                                                                                                                                                              
+---- | ---- | ---- | :----: |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Vertriebs-Rohdaten-Report | ```/rohdaten``` | `report:rohdaten:lesen`  | zip/UTF-8 | all relevant data of [Vorgänge, Anträge, Bausteine](https://docs.api.europace.de/common/glossary/) and [Provisionen](https://docs.api.europace.de/common/glossary/) of the advisor<br>Data older than 2014 will not be delivered. |
-Produktanbieter-Report | ```/produktanbieter``` | `report:produktanbieter:lesen`  | csv/UTF-8 | the essential data of [Anträge](https://docs.api.europace.de/common/glossary/) with state and [Vertriebsorganisation](https://docs.api.europace.de/common/glossary/) |
-
+Produktanbieter-Report | ```/produktanbieter``` | `report:produktanbieter:lesen`  | csv/UTF-8 | the essential data of [Anträge](https://docs.api.europace.de/common/glossary/) with state and [Vertriebsorganisation](https://docs.api.europace.de/common/glossary/)                                                              |
+Vertriebsreport | ```/vertrieb``` | ```report:rohdaten:lesen``` | csv/UTF-8 | the "EUROPACE Report Vertrieb" from Partnermanagement.                                                                                                                                                                            | 
```

Comment:
I would leave out "from Partnermanagement" because this is an implementation detail.

Reply:
Fixed in [2040463](https://github.com/europace/report-api/pull/17/commits/2040463c80066469ba6969178d79f0ee64a14dcc).